weather.py

```diff
@@ -271,15 +272,15 @@ def __del__(self):
         "Destructor to make sure pygame shuts down, etc."
 
     def get_forecast(self):
-        if (time.time() - self.last_update_check) > config.DS_CHECK_INTERVAL:
+        if (time.time() - self.last_update_check) > int(CONFIG["update_freq"]):
             self.last_update_check = time.time()
             try:
-                self.weather = forecast(config.DS_API_KEY,
-                                        config.LAT,
-                                        config.LON,
+                self.weather = forecast(CONFIG["api_key"],
```

Comment:
Please keep the ds prefex or some other identifier. I need to add a second api key into the mix soon so they will need to be able to be told apart.

Reply:
good idea.